sklearn/linear_model/tests/test_base.py

```diff
@@ -27,7 +27,6 @@
 from sklearn.preprocessing import StandardScaler
 from sklearn.preprocessing import add_dummy_feature
 
-rng = np.random.RandomState(0)
```

Comment:
obsolete, handled with new fixture `global_random_seed` inside functions

Reply:
Now that we remove the `global_random_seed` from a few functions where it is not needed, maybe we can pick up this global `rng` again @glemaitre ?

i.e.:
```python
rng = check_random_state(0)
```